.env.example

```
@@ -0,0 +1,7 @@
+# RabbitMQ container credentials (used by docker-compose for RabbitMQ service)
+ RABBITMQ_USER=your_username
```

dotenv-linter (3.3.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)